src/models/pipeline-run-task-step/test-data.ts

```diff
@@ -0,0 +1,46 @@
+import { PipelineRunTaskStepModel } from '.';
+
+import { TestDependencyDataType } from '../model-types';
+import { ProgressStatus } from '../progress-status';
+
+/*
+ * TestData array
+ *
+ * Note: First object should always be valid
+ */
+export const testData: TestDependencyDataType<PipelineRunTaskStepModel> = [
+  {
+    __testDescription: 'Valid full object',
+    name: 'some-task-step',
+    started: new Date('2022-05-10T14:31:23Z'),
+    ended: new Date(),
+    status: ProgressStatus.Succeeded,
```

Comment:
add missing optional object in 'Valid full object' test
```suggestion
    status: ProgressStatus.Succeeded,
    statusMessage: 'statusMessage',
```